### ISSUE TITLE
Structs3: relies on panic! which is not introduced in the official book until chapter 9

### ISSUE THREAD

**Issue by bitmingw** [rust-lang/rustlings#685]
The exercise of Structs3 requires knowledge of `panic!` to complete the task.

```rust
    fn new(sender_country: String, recipient_country: String, weight_in_grams: i32) -> Package {
        if weight_in_grams <= 0 {
            // Something goes here...
        } else {
            return Package {
                sender_country,
                recipient_country,
                weight_in_grams,
            };
        }
    }
```

However `panic!` is not introduced in the official book until chapter 9 (Error Handling), so user who closely follows the book and does exercise will have additional challenge to find the right answer here.

Maybe we should just provide the `panic!` implementation here instead of asking user to find it?

**Comment by carneeki** [comment]
Agree, having the user implement the new package (and perhaps seeing the `panic!` but not needing to implement it) is probably more inline with a lesson about structs.

**Comment by VMois** [comment]
Agree. Needed to search how to `panic!`. Would be good to introduce panic earlier or to provide an implementation for it (the second option sounds better).

**Comment by thephet** [comment]
Hi, I am doing the exercises as a noob, and this needs to be fixed. No idea what I am supposed to do.
